types/mui-datatables/index.d.ts

```diff
@@ -84,24 +83,24 @@ interface MUIDataTableTextLabelsViewColumns {
 }
 
 interface MUIDataTableTextLabelsSelectedRows {
-    text: string;
     delete: string;
     deleteAria: string;
+    text: string;
```

Comment:
Why is this re-ordered?

Reply:
Alphabetized properties inside interface